Pathogens/Utility/location_utils/country_iso2.json

```diff
@@ -210,6 +211,7 @@
   "Switzerland": "CH",
   "Syrian Arab Republic": "SY",
   "Tajikistan": "TJ",
+  "Taiwan": "TW",
```

Comment:
Contained in airtable but not in country code files, you can check Wikipedia if you want to make sure I'm not making these up lol